websocketOutportDriver/client/wsClientFactory.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"github.com/multiversx/mx-chain-core-go/data/typeConverters/uint64ByteSlice"
+	"github.com/multiversx/mx-chain-core-go/websocketOutportDriver"
+)
+
+// ArgsCreateWsClient is a placeholder struct for args required to create a WSClient
+type ArgsCreateWsClient struct {
+	Url                string
+	RetryDurationInSec uint32
+	BlockingAckOnError bool
+	PayloadProcessor   PayloadProcessor
+}
+
+// CreateWsClient creates a WSClient
+func CreateWsClient(args ArgsCreateWsClient) (WSClient, error) {
```

Comment:
we need unit tests for this factory method. 
You can use something like
```GO
assert.Equal(t, "*websocketOutportDriver.webSocketPayloadParser", fmt.Sprintf("%T", instance)
```
to test that the type is really that expected type.